src/redux/explorer.js

```diff
@@ -77,8 +87,8 @@ const addressSubscription = (address, currency, action = 'subscribe') => [
   },
 ];
 
-const assetsSubscription = (pairs, currency, action = 'subscribe') => {
-  const assetCodes = concat(keys(pairs), 'eth');
+const assetPricesSubscription = (pairs, currency, action = 'subscribe') => {
```

Comment:
can we have `pairs` argument just be an array of `tokenAddresses` instead (and remove `keys(pairs)` below - and update functions that call this to pass in the keys directly (they can do keys(whatever) when they call the function)